urllib3/connection.py

```diff
@@ -62,6 +57,14 @@ class ConnectionError(Exception):
 RECENT_DATE = datetime.date(2014, 1, 1)
 
 
+class DummyConnection(object):
+    "Used to detect a failed ConnectionCls import."
+    pass
+
+
+HTTPSConnection = DummyConnection
```

Comment:
Agreed.

I must admit, I am not thrilled with the `HTTPSConnection` class being hidden at the end of the module.  Any objection to the `HTTPSConnection` class being renamed to `_HTTPSConnection` or `UnverifiedHTTPSConnection`?  Or simply merged it into `VerifiedHTTPSConnection`.

I could do that as a separate earlier commit as part of this pull request.